CONTRIBUTORS.md

```diff
@@ -10,7 +10,7 @@
 | **Matteo Collina**   | [**@mcollina**](https://github.com/mcollina)           | [**@matteocollina@twitter**](https://twitter.com/matteocollina)     |
 | **James Halliday**   | [**@substack**](https://github.com/substack)           | [**@substack@twitter**](https://twitter.com/substack)               |
 | **David Björklund**  | [**@kesla**](https://github.com/kesla)                 | [**@david_bjorklund@twitter**](https://twitter.com/david_bjorklund) |
-| **Jake Verbaten**    | [**@Raynos**](https://github.com/Raynos)               | [**@raynos@twitter**](https://twitter.com/raynos)                   |
+| **Raynos**           |                                                        |                                                                     |
```

Comment:
This is caused by some change between `remark-git-contributors` 0.2.1 and 0.3.0, but IMO it's not important to fix now.